# scipy.stats.DegenerateDataWarning

> 原文：[https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.stats.DegenerateDataWarning.html#scipy.stats.DegenerateDataWarning](https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.stats.DegenerateDataWarning.html#scipy.stats.DegenerateDataWarning)

```py
exception scipy.stats.DegenerateDataWarning(msg=None)
```

在数据退化且结果可能不可靠时发出警告。

```py
with_traceback()
```

Exception.with_traceback(tb) – 将 self.__traceback__ 设置为 tb 并返回 self。
